### DISCUSSION TITLE
Question about geometry optimisations

### DISCUSSION THREAD

**Discussion by jennijones** [rdkit/rdkit#6376] [Q&A] [answered]
When I run MMFFOptimizeMolecule on a molecule and it is unsuccessful, I can run it again either once more or several more times until it is eventually successful. Is this random, or is this equivalent to increasing the maximum number of iterations?

**Comment by ptosco** [comment, marked as answer]
@jennijones `MMFFOptimizeMolecule` will return `1` if it has reached `maxIters` without finding a local minimum, otherwise it will return `0`:
```
from rdkit import Chem
from rdkit.Chem import AllChem

stringy_mol = Chem.AddHs(Chem.MolFromSmiles("CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"))
AllChem.EmbedMolecule(stringy_mol)
0

AllChem.MMFFOptimizeMolecule(stringy_mol)
1

AllChem.MMFFOptimizeMolecule(stringy_mol, maxIters=100000)
0
```

**Comment by UnixJunkie** [comment]
this could be marked as properly answered
